### PR TITLE
login and sign up 

### PR DESCRIPTION
**Pull request by Rajitha04**
Added Login and signup pages.
Please review, Thanks!
login:https://1gto05.csb.app/components/login.html
signup:https://1gto05.csb.app/components/signup.html

### CODE REVIEW

**Review by rahul5564 — APPROVED** [review]
Add header and footer in login and signup page

File: components/signup.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+body{
+    max-width: 400px;
+    margin: auto;
+    margin-top: 4rem;
+    margin-bottom: 4rem;
+}
+input{
+    padding: 1rem;
+    border-radius: 0.5rem;
+    width: 350px;
```
Comment (srv-pranjal) [inline components/signup.css:10]: At some places you've used rems and at others pixels, Avoid using different types of units.

File: components/signup.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    padding: 1rem;
+    border-radius: 0.5rem;
+    background-color: rgb(207, 87, 87);
+    margin-top: 1rem;
+    width: 175px;
+}
+.signup{
+    padding: 1rem;
+    border-radius: 0.5rem;
+    background-color: rgb(115, 177, 115);
```
Comment (srv-pranjal) [inline components/signup.css:25]: Use variables for colors.

**Review by srv-pranjal — COMMENTED** [review]
Suggested few things, rest everything looks good. Also if there isn't any reason to include CL files in this repo, then remove them.

**Pull request merged**